File: Easy/125_valid_palindrome.cpp
class Solution {
public:
    bool isPalindrome(string s) {
        string b;
        for(auto g: s){
            if(g > 64 && g <= 90){
                b.push_back(g+32);
                continue;
            }
            if(g > 96 && g <= 122 || g > 47 && g < 58){
                b.push_back(g);
                continue;
            }
        }
        int k = b.length()-1;
        for( int i = 0; i <= k; i++){
            if( b[i] != b[k]) return false;
            k--;
        }
        return true;
    }
};

File: python/easy/125_valid_palindrome.py
class Solution:
    def isPalindrome(self, s: str) -> bool:
        copy = str()
        reverse = str()
        for char in s:
             if char.isalpha() or char.isdigit():
                copy+=char.lower()
        for i in range(len(copy) -1, -1, -1):
            reverse+=copy[i]
        return reverse == copy            
